ee/fe/modules/340-extended-monitoring/openapi/config-values.yaml

```diff
@@ -52,6 +52,12 @@ properties:
         default: false
         description: |
           Enables certExporter.
+      namespaces:
```

Comment:
I have a proposal to stick to k8s-image-availability-exporter here:
1. Always collect metrics from all namespaces.
2. Alert about expiring certificates only if the extended monitoring enabled label present on the namespace.

Thus we could avoid adding the `namespaces` option to the config.

Reply:
OK, solved in https://github.com/deckhouse/deckhouse/pull/479/commits/6402266d7eb82226ee1f84925321b277a965e095